test/e2e/manage_existing_resources_test.go

```diff
@@ -0,0 +1,100 @@
+	"k8s.io/apimachinery/pkg/util/rand"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/klog/v2"
+
+	policyv1alpha1 "github.com/karmada-io/karmada/pkg/apis/policy/v1alpha1"
+	workv1alpha2 "github.com/karmada-io/karmada/pkg/apis/work/v1alpha2"
+	"github.com/karmada-io/karmada/test/e2e/framework"
+	"github.com/karmada-io/karmada/test/helper"
+)
+
+var _ = ginkgo.Describe("manage existing resources testing", func() {
+
+	ginkgo.Context("manage existing core resources testing", func() {
+		policyNamespace := testNamespace
+		policyName := deploymentNamePrefix + rand.String(RandomStrLength)
+		deploymentNamespace := testNamespace
+		deploymentName := policyName
+
+		deployment := helper.NewDeployment(deploymentNamespace, deploymentName)
+		policy := helper.NewPropagationPolicy(policyNamespace, policyName, []policyv1alpha1.ResourceSelector{
+			{
+				APIVersion: deployment.APIVersion,
+				Kind:       deployment.Kind,
+				Name:       deployment.Name,
+			},
+		}, policyv1alpha1.Placement{
+			ClusterAffinity: &policyv1alpha1.ClusterAffinity{
+				ClusterNames: framework.ClusterNames(),
+			},
+		})
+
+		ginkgo.It("manage existing Deployments testing", func() {
+			member1Client := framework.GetClusterClient(framework.ClusterNames()[0])
+			if _, err := member1Client.CoreV1().Namespaces().Get(context.TODO(), testNamespace, metav1.GetOptions{}); apierrors.IsNotFound(err) {
+				framework.CreateNamespace(member1Client, helper.NewNamespace(testNamespace))
+			}
+			framework.CreateDeployment(member1Client, deployment)
+
+			framework.CreatePropagationPolicy(karmadaClient, policy)
+			framework.CreateDeployment(kubeClient, deployment)
+
+			ginkgo.By("check deployment's replicas", func() {
+				wantedReplicas := *deployment.Spec.Replicas * int32(len(framework.Clusters())-1)
+
+				klog.Infof("Waiting for deployment(%s/%s) collecting status", deploymentNamespace, deploymentName)
+				err := wait.PollImmediate(pollInterval, pollTimeout, func() (done bool, err error) {
+					currentDeployment, err := kubeClient.AppsV1().Deployments(testNamespace).Get(context.TODO(), deploymentName, metav1.GetOptions{})
+					gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
+
+					klog.Infof("deployment(%s/%s) readyReplicas: %d, wanted replicas: %d", deploymentNamespace, deploymentName, currentDeployment.Status.ReadyReplicas, wantedReplicas)
+					if currentDeployment.Status.ReadyReplicas == wantedReplicas &&
+						currentDeployment.Status.AvailableReplicas == wantedReplicas &&
+						currentDeployment.Status.UpdatedReplicas == wantedReplicas &&
+						currentDeployment.Status.Replicas == wantedReplicas {
+						return true, nil
+					}
+
+					return false, nil
+				})
+				gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
+			})
+
+			annotations := map[string]string{workv1alpha2.ResourceConflictResolutionAnnotation: workv1alpha2.ResourceConflictResolutionOverwrite}
+			framework.UpdateDeploymentAnnotations(kubeClient, deployment, annotations)
```

Comment:
Maybe we can add a log here to indicate that take-over deployment on member cluster.

Reply:
Done